ClosedXML.Examples/.editorconfig

```diff
@@ -0,0 +1,5 @@
+# C# files
+[*.cs]
+
+# S1481: Unused local variables should be removed
+dotnet_diagnostic.S1481.severity = none
```

Comment:
are there so many that it would hurt removing those?

Reply:
Unused local variables are used to explain the code.